pkg/statistics/store_load.go

```diff
@@ -272,20 +277,24 @@ func NewStoreHistoryLoads(dim int) *StoreHistoryLoads {
 }
 
 // Add adds the store load to the history.
-func (s *StoreHistoryLoads) Add(storeID uint64, rwTp utils.RWType, kind constant.ResourceKind, loads []float64) {
+func (s *StoreHistoryLoads) Add(storeID uint64, rwTp utils.RWType, kind constant.ResourceKind, pointLoad []float64) {
 	load, ok := s.loads[rwTp][kind][storeID]
 	if !ok {
-		size := defaultSize
-		if historySampleInterval != 0 {
-			size = int(HistorySampleDuration / historySampleInterval)
+		size := int(DefaultHistorySampleDuration / DefaultHistorySampleInterval)
+		if s.sampleInterval != 0 {
+			size = int(s.sampleDuration / s.sampleInterval)
```

Comment:
should we hard code a fixed ratio as size rather than supporting a interval? maybe can reduce some configuration